app/models/ThriftMe.sql

```diff
@@ -29,6 +29,8 @@ CREATE TABLE photos (
     photo_data BYTEA,
     FOREIGN KEY (album_id) REFERENCES albums(album_id) ON DELETE CASCADE
 );
+ALTER TABLE photos
+ALTER COLUMN photo_data TYPE TEXT;
```

Comment:
we can probs get rid of these two lines

Reply:
Ohh gotcha so that's just for ur end?

Reply:
Yeah, since you have your database running now you can just ignore it and it won't change anything, or use it to switch the table if needed.